uqcs/templates/form.mako

```diff
@@ -152,7 +157,7 @@
             id="degreeInput"
             name="degreeInput"
             class="form-control"
-            placeholder="Bachelor of Engineering"
+            placeholder="Search with keywords, e.g. &quot;math, engineering&quot;"
```

Comment:
Perhaps compsci would be a more appropriate first suggestion?

Reply:
A problem is the input field becomes narrower on mobile, which is why I went with a shorter first keyword. I'll see what else there is.